_plugins/jekyll_get.rb

```diff
@@ -14,8 +14,8 @@ def generate(site)
       end
 
       # slurp down code.json data
-      url = site.config['scanner_url'] + 'search200/json/?200page=/code.json&present=Present'
-      data = JSON.load(open(url))
+      url = site.config['scanner_url'] + 'search200/json/?200page=/data.json&present=Present&mimetype=application/json'
+      data = JSON.load(URI.open(url))
```

Comment:
I'm not that familiar with Ruby, but can you tell me about this change? Was the URI library not in scope before?

Reply:
I was getting this warning during build:
`...18f/site-scanning-dashboard/_plugins/jekyll_get.rb:18: warning: calling URI.open via Kernel#open is deprecated, call URI.open directly or use URI#open`